lib/govuk/components/engine.rb

```diff
@@ -113,6 +113,7 @@ def reset!
 
       default_link_new_tab_text: "(opens in new tab)",
 
+      error_prefix: "Error: ",
```

Comment:
Ideally there'd be a line in the documentation block above explaining what this does.

Reply:
@peteryates done. I've also renamed it to `default_error_prefix` for consistency.

There’s currently not any way to override it within the helper itself though. Wonder if there should be? Maybe that's something to consider down the line if/when multiple languages are supported?

Reply:
Elsewhere I've set values like this in the following manner which is concise and seems to have worked well.

```ruby
  def title_with_error_prefix(title, error:, error_prefix: Govuk::Components.config.default_error_prefix)
    ...

```

Reply:
Ah, nice. Done in 574de503d5024c06cc0a1911641a0c7ee9b4a24f